xbmc/network/DNSNameCache.cpp

```diff
@@ -54,7 +54,7 @@ bool CDNSNameCache::Lookup(const std::string& strHostName, std::string& strIpAdd
   if(g_DNSCache.GetCached(strHostName, strIpAddress))
     return true;
 
-#ifndef TARGET_WINDOWS
+#if !defined(TARGET_WINDOWS) && defined(HAVE_SMBCLIENT)
```

Comment:
still wrong name

Reply:
I'm sorry, I misunderstood how defines worked between cmake and the c defines. It took me a bit of research and trial and error to get there (especially since an incorrect value doesn't cause a build failure, but an unexpected runtime result for something I don't personally use). I greatly appreciate your review and patience, thank you again.